scripts/test-e2e-local.js

```diff
@@ -218,52 +214,51 @@ async function testRNTestProject(circleCIArtifacts) {
   );
 
   updateTemplatePackage({
-    'react-native': `file:${localNodeTGZPath}`,
+    'react-native': `file://${localNodeTGZPath}`,
   });
 
-  // create locally the node module
-  exec('npm pack --pack-destination ', {cwd: reactNativePackagePath});
-
-  // node pack does not creates a version of React Native with the right name on main.
-  // Let's add some defensive programming checks:
-  if (!fs.existsSync(localNodeTGZPath)) {
-    const tarfile = fs
-      .readdirSync(reactNativePackagePath)
-      .find(name => name.startsWith('react-native-') && name.endsWith('.tgz'));
-    if (!tarfile) {
-      throw new Error("Couldn't find a zipped version of react-native");
-    }
-    exec(
-      `cp ${path.join(reactNativePackagePath, tarfile)} ${localNodeTGZPath}`,
-    );
-  }
-
   pushd('/tmp/');
   // need to avoid the pod install step - we'll do it later
   exec(
-    `node ${reactNativePackagePath}/cli.js init RNTestProject --template ${localNodeTGZPath} --skip-install`,
+    `node ${reactNativePackagePath}/cli.js init RNTestProject --template ${reactNativePackagePath} --skip-install`,
   );
 
   cd('RNTestProject');
   exec('yarn install');
 
+  // When using CircleCI artifacts, the CI will zip maven local into a
+  // /tmp/maven-local subfolder struct.
+  // When we generate the project manually, there is no such structure.
+  const expandedMavenLocal =
+    circleCIArtifacts == null
+      ? mavenLocalPath
+      : `${mavenLocalPath}/tmp/maven-local`;
   // need to do this here so that Android will be properly setup either way
   exec(
-    `echo "REACT_NATIVE_MAVEN_LOCAL_REPO=${mavenLocalPath}" >> android/gradle.properties`,
+    `echo "react.internal.mavenLocalRepo=${expandedMavenLocal}" >> android/gradle.properties`,
```

Comment:
Not sure about this line, tbh.
I don't think that Nico backported this property to 0.72. 🤔

Reply:
That makes sense, I'll revert this specific change then 👍